requirements.txt

```diff
@@ -3,9 +3,9 @@ psutil
 ray >= 2.9
 sentencepiece  # Required for LLaMA tokenizer.
 numpy
-torch == 2.1.2
+torch == 2.2.0
 transformers >= 4.38.0  # Required for Gemma.
-xformers == 0.0.23.post1  # Required for CUDA 12.1.
+xformers == 0.0.24  # Required for PyTorch 2.2.0.
```

Comment:
I think the original comment was referring to the version.

I was asked to update it in an earlier review because `PyTorch 2.2.0` requires `0.0.24`, not `CUDA 21.1`

Reply:
I mean, I don't see `xformers` in pytorch's https://github.com/pytorch/pytorch/blob/main/requirements.txt .

Do you mean `xformers == 0.0.24` requires `pytorch 2.2.0`?

Reply:
> Do you mean `xformers == 0.0.24` requires `pytorch 2.2.0`?

Yeah, that's what the comment means.

And the original comment meant that `xformers==0.0.23.post1` required `CUDA 12.1`, not the other way round.

Reply:
Ok, I'm happy with that change

```suggestion
xformers == 0.0.24  # Requires PyTorch 2.2.0.
```